src/Visitor/Php/Symfony/ValidationAnnotation.php

```diff
@@ -73,7 +73,7 @@ public function enterNode(Node $node)
             /** @var ClassMetadata $metadata */
             $metadata = $this->metadataFactory->getMetadataFor($name);
         } catch (AnnotationException $e) {
-            $this->addError($node, 'Could not parse class "%s" for annotations. %s', $this->namespace, $e->getMessage());
```

Comment:
I think ```$this->namespace``` and ```$e->getMessage``` can be merged with previous string.

Reply:
Ahh I am not thinking! Thanks!